tests/foreman/installer/test_installer.py

```diff
@@ -1306,10 +1306,30 @@ def extract_help(filter='params'):
 def common_sat_install_assertions(satellite):
     sat_version = 'stream' if satellite.is_stream else satellite.version
     assert settings.server.version.release == sat_version
+
+    # no errors/failures in /var/log/messages
+    # ignore warnings (some messages contain string "error" but they're still only warnings)
+    # ignore "kernel: GPT: Use GNU Parted to correct GPT errors."
+    result = satellite.execute(
+        r'grep -i "error" /var/log/messages | grep -v "\] WARNING" | grep -v GPT'
+    )
```

Comment:
Seems like the test failed here. Did you come up with these assertions? Did support mention the coverage here wasn't good enough?

Reply:
The idea to check syslog for errors was there in a stubbed testcase. I finally implemented it. But it returned lots of false positives. For example error messages of applications not related to foreman/satellite that were logged even before the installation.  Or some minor warnings and info messages containg the word "error". So instead of syslog I decided to use journalctl which allows advanced filtering - see my next commit. Suggestions to adjust the filtering are welcome - we can implement them within this PR or later.

Reply:
And latest PRT finished successfully :-)